src/resources/views/crud/form_content.blade.php

```diff
@@ -48,6 +48,58 @@ function initializeFieldsWithJavascript(container) {
       });
     }
 
+    /**
+     * Auto-discover first focusable input
+     * @param {jQuery} form
+     * @return {jQuery}
+     */
+    function getFirstFocusableField(form) {
+        return form.find('input, select, textarea, button')
+            .not('.close')
+            .not('[readonly]')
```

Comment:
from my understanding `readonly` fields should be focusable right ? at least the `text` ones since they allow you to copy the value. 

I know it's debatable here, and maybe the name `getFirstFocusableField` is misleading because a readonly text field would be a perfectly valid `focusable` input I guess. 

If we really want to skip "un-editable" inputs we should probably rename the function to `getFirstEditableInput` and `triggerFocusOnFirstEditableInput`. I think they better describe what we are doing here.

Also @tabacitu what are your expectations about readonly fields? Imagine this: 
![image](https://user-images.githubusercontent.com/7188159/179200007-89d33fe8-cf44-4662-8a62-b8b4d281f59b.png)

Would you expect the readonly field to be focused and then you tab away, or select text and copy, whatever, or should we move the mindset into `first editable field`, so `slug` would be the focused input ?

Reply:
Hmm... this is a tricky one. I agree with both of you. Great explanation and name @pxpm 👏👏👏 

I don't see a perfect solution here, so this just needs an executive decision.

Because:
- I see this use case as very very rare and improbable (first input is readonly);
- I don't want this feature to spiral out of control and become a rewrite/refactor;

Let's 
- keep the name as `getFirstFocusableField`;
- make the feature logic follow the feature name (readonly inputs are focusable);

Again... this use case with the first input as focusable... very improbable. So let's not overthink this.